cli/pcluster/configure/networking.py

```diff
@@ -193,23 +192,21 @@ def _validate_cidr(cidr):
 
 
 @handle_client_exception
-def get_vpc_subnets(aws_region_name, vpc_id):
+def get_vpc_subnets(vpc_id):
```

Comment:
I think you forgot to update L54, L55 and L56 where the region name is still being passed to the function

Reply:
Fixed, thank you